python/cuml/test/dask/test_ridge_regression.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+from dask_cuda import LocalCUDACluster
+
+from dask.distributed import Client
+from sklearn.metrics import mean_squared_error
+import pandas as pd
+import gzip
+import numpy as np
+import os
+
+pytestmark = pytest.mark.mg
+
+
+def load_data(nrows, ncols, cached='data/mortgage.npy.gz'):
```

Comment:
Why load data instead of using make_regression? Make_regression seems faster and more customizable.

Reply:
Used this data in the single GPU case as well. That function can also generate random numbers.

Reply:
Done.